runatlantis.io/blog.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Welcome to Our Blog
+aside: false
+---
+
+# Welcome to Our Blog
+
+We are thrilled to have you here! Our blog is a collection of insightful articles, tips, and updates from our team. Whether you're new or have been following us for a while, there's always something new to learn and explore.
+
+### Explore Our Popular Posts
+
+We have a rich history of blog posts dating back to 2017-2019. Here are some of our popular posts:
+
+- [4 Reasons To Try HashiCorp's (New) Free Terraform Remote State Storage](/blog/4-reasons-to-try-hashicorps-new-free-terraform-remote-state-storage)
+- [I'm Joining HashiCorp!](/blog/joining-hashicorp)
+- [Putting The Dev Into DevOps: Why Your Developers Should Write Terraform Too](/blog/putting-the-dev-into-devops-why-your-developers-should-write-terraform-too)
+- [Atlantis 0.4.4 Now Supports Bitbucket](/blog/atlantis-0-4-4-now-supports-bitbucket)
+- [Terraform And The Dangers Of Applying Locally](/blog/terraform-and-the-dangers-of-applying-locally)
+- [Hosting Our Static Site over SSL with S3, ACM, CloudFront and Terraform](/blog/hosting-our-static-site-over-ssl-with-s3-acm-cloudfront-and-terraform)
+- [Introducing Atlantis](/blog/introducing-atlantis)
+
+### Welcoming New Blog Authors
+
+We are excited to welcome new authors to our blog. Our diverse team brings a wealth of knowledge and experience to share with our readers. Stay tuned for fresh perspectives and in-depth articles on the latest trends and technologies.
+
+If you have any questions or topics you would like us to cover, feel free to reach out. We are always looking to engage with our community and provide valuable content.
```

Comment:
worth linking to a page with ways to reach out (or perhaps Slack community link; if that is preferred), so we have "call to action" and "action" tied neatly together

Reply:
yeah, good point